homeassistant/components/alarm_control_panel/manual.py

```diff
@@ -24,6 +24,11 @@
 DEFAULT_TRIGGER_TIME = 120
 DEFAULT_DISARM_AFTER_TRIGGER = False
 
+PENDINGABLE_STATES = [STATE_ALARM_ARMED_AWAY,
```

Comment:
This word sounds weird. Can we find a better one? `STATES_PENDING`?

Reply:
How about `PENDING_SUPPORTED_STATES`?

Reply:
`SUPPORTED_PENDING_STATES`